prisma/schema.prisma

```diff
@@ -0,0 +1,85 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Account {
+  id                 String  @id @default(cuid())
+  userId             String
+  type               String
+  provider           String
+  providerAccountId  String
+  refresh_token      String?  @db.Text
+  access_token       String?  @db.Text
+  expires_at         Int?
+  token_type         String?
+  scope              String?
+  id_token           String?  @db.Text
+  session_state      String?
+
+  user             User   @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: NoAction)
+
+  @@unique([provider, providerAccountId])
+}
+
+model AdminLog {
+  id          Int      @id @default(autoincrement())
+  userId     String
+  timestamp   DateTime @default(now()) @db.Timestamptz(6)
+  action      String   
+  event       String   
+  description String?
+  user       User    @relation(fields: [userId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+}
+
+model FormUser {
+  id              String       @id @default(cuid())
+  templateId     String
+  email           String    @db.VarChar(80)
+  temporaryToken String?
+  active          Boolean   @default(true)
+  created_at      DateTime  @default(now()) @db.Timestamptz(6)
+  updated_at      DateTime  @default(now()) @db.Timestamptz(6)
```

Comment:
I wonder why you choose to set  `updated_at` field  default value to `now()` and make it required?  
Since  `updated_at` is not required when creating the entity , I think it should be marked as optional and only updatable. Like  `updated_at    DateTime? @updatedAt`

Reply:
Good catch!  It was an oversight after the introspection of the existing Postgres Database Scheme.
I'll modify it to use the built-in `@updatedAt` Prisma attribute for the field.  I don't think the field needs to be marked as optional when set with the `@updatedAt` attribute but I'll definitely look into it.